examples/android/compass/main.py

```diff
@@ -20,14 +20,15 @@
 
 
 import kivy
-kivy.require('1.7.0')
```

Comment:
this is in the beginning exactly for the reason it might seem obvious - to kill the app if there's no minimal required version. Again, broken, because the rest of imports were triggered and _then_ the version requirement might fail. In this case, however, it might be probably better to completely remove it. `1.7.0` is ancient and this line does nothing at all now.